app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapFragment.java

```diff
@@ -316,32 +370,15 @@ private void initViews() {
     }
 
     /**
-     * a) Creates bottom sheet behaviours from bottom sheet, sets initial states and visibility
-     * b) Gets the touch event on the map to perform following actions:
-     *      if bottom sheet details are expanded or collapsed hide the bottom sheet details.
+     * a) Creates bottom sheet behaviours from bottom sheet, sets initial states and visibility b)
```

Comment:
I think this javadoc was more readable before.

Reply:
Sure and Thanks !, Sorry I did not see that. Shared a pull request https://github.com/commons-app/apps-android-commons/pull/5481